Converters/Revit/Speckle.Converters.RevitShared/ToSpeckle/Raw/MaterialQuantitiesToSpeckle.cs

```diff
@@ -0,0 +1,130 @@
+using Speckle.Converters.Common.Objects;
+using Speckle.Converters.RevitShared.Helpers;
+using Speckle.Converters.RevitShared.Services;
+using Speckle.Objects.Other;
+using Speckle.Objects.Other.Revit;
+
+namespace Speckle.Converters.RevitShared.ToSpeckle;
+
+public class MaterialQuantitiesToSpeckleLite : ITypedConverter<DB.Element, List<Dictionary<string, object>>>
+{
+  private readonly ScalingServiceToSpeckle _scalingService;
+  private readonly IRevitConversionContextStack _contextStack;
+
+  public MaterialQuantitiesToSpeckleLite(
+    ScalingServiceToSpeckle scalingService,
+    IRevitConversionContextStack contextStack
+  )
+  {
+    _scalingService = scalingService;
+    _contextStack = contextStack;
+  }
+
+  /// <summary>
+  ///
+  /// </summary>
+  /// <param name="target"></param>
+  /// <returns></returns>
```

Comment:
nice doc
jokes aside, it might be good to explain why it is list of dict as key string as value object?